apps/theming/css/theming.scss

```
@@ -14,12 +14,18 @@
 @mixin faded-background-image {
 	@if ($color-primary == #0082C9) {
 		background-image: $image-login-background, linear-gradient(40deg, $color-primary 0%, lighten($color-primary, 20%) 100%);
+
+		@if($has-custom-background == true) {
```

Tiny nitpick but this should work the same as 
```suggestion
		@if $has-custom-background {
```